### ISSUE TITLE
New expanding search box on juju.is too?

### ISSUE THREAD

**Issue by tmihoc** [canonical/juju.is#442]
@bartaz in https://ubuntu.com/blog/release-of-vanilla-framework-v3#expanding-search-box you were announcing a new expanding search box. As you mentioned there, this has already been implemented on https://ubuntu.com/ . Is there any way we can get it on juju.is too?

**Comment by bartaz** [comment]
Yes, it's [part of Vanilla](https://vanillaframework.io/docs/patterns/navigation#expanding-search-box), can be added to top navigation of any site that needs that.

As far as I can see the juju.is currently only has search for docs. Do you want this search to be placed in top nav and expandable? https://juju.is/docs

**Comment by tmihoc** [comment]
Yes. We'd like the solution implemented for https://ubuntu.com/server/docs (there's the expandable site search in the top right of the menu and then also the large docs search just beneath the menu).

**Comment by beliaev-maksim** [comment]
> Yes, it's [part of Vanilla](https://vanillaframework.io/docs/patterns/navigation#expanding-search-box), can be added to top navigation of any site that needs that.
> 

Sounds like a simple configuration tweak, is it?
If yes, then please go ahead

**Comment by Lukewh** [comment]
Currently, this isn't feasible with the separation of charmhub and juju, and having a shared navigation. On charmhub that search is used to search charms/bundles, on juju it would be to search docs. @AnnSoMuller has been working on a solution for a unified search: One that has a single search in the top right and provides results from docs, discourse, and charms/bundles in a single interface.
The work is currently in the UX and design phase, but we should be in a position to implement it next cycle :).
